admin_manual/installation/installation_wizard.rst

```diff
@@ -175,3 +175,7 @@ and files:
 * The :file:`data/.htaccess` file should be owned by ``root:[HTTP group]``
 * Both :file:`.htaccess` files are read-write file owner, read-only group and 
   world
+
+In case you want to update your installation via the GUI or the occ-command,
+check the **Setting Strong Directory Permissions** in the Update section.
```

Comment:
s/check the/check/
"the" is not needed before proper nouns (that are essentially a "name"), whereas it is correct to say "the Update section" because "section" is an ordinary noun.